modules/node-shared/src/main/scala/org/tessellation/node/shared/domain/security/DoubleSignDetection.scala

```diff
@@ -0,0 +1,16 @@
+package org.tessellation.node.shared.domain.security
+
+import org.tessellation.schema.SnapshotOrdinal
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.security.hash.Hash
+
+import derevo.cats.{eqv, show}
+import derevo.derive
+
+@derive(eqv, show)
+case class DoubleSignDetection(
```

Comment:
maybe `DetectedDoubleSign`?

Reply:
Renamed to `DetectedDoubleSign` and moved to `domain.snapshot`.